Formula/g/git-annex.rb

```diff
@@ -23,15 +23,8 @@ class GitAnnex < Formula
   depends_on "libmagic"
 
   resource "bloomfilter" do
-    url "https://hackage.haskell.org/package/bloomfilter-2.0.1.0/bloomfilter-2.0.1.0.tar.gz"
-    sha256 "6c5e0d357d5d39efe97ae2776e8fb533fa50c1c05397c7b85020b0f098ad790f"
-
-    # Fix build with GHC >= 9.2
-    # PR ref: https://github.com/bos/bloomfilter/pull/20
-    patch do
-      url "https://github.com/bos/bloomfilter/commit/fb79b39c44404fd791a3bed973e9d844fb084f1e.patch?full_index=1"
-      sha256 "c91c45fbdeb92f9dcb9b55412d14603b4e480139f6638e8b6ed651acd92409f3"
-    end
+    url "https://hackage.haskell.org/package/bloomfilter-2.0.1.2/bloomfilter-2.0.1.2.tar.gz"
+    sha256 "178293d28a922e89eddffa597513cb2287231322fc8834765762f0d9a8118b4e"
```

Comment:
New release may allow us to just remove resource and workaround inside install block

Reply:
yeah, that sounds about right.